_data/simple-icons.json

```diff
@@ -12526,6 +12526,12 @@
             "hex": "4695EB",
             "source": "https://design.jboss.org/quarkus"
         },
+        {
+            "title": "Quarto",
+            "hex": "39729E",
+            "source": "https://quarto.org",
```

Comment:
Only seeing a PNG at this source... we need an SVG source here.

Reply:
The SVG I used is only hosted internally as far as I can tell. The only thing close is this icon: https://github.com/quarto-dev/quarto/blob/83a2dda63d7f2de7239654fc671f48e9c3f53b81/apps/vscode/assets/icon/qmd.svg?plain=1 

Would that be better to link to?

Reply:
If it lines up with the icon in this PR, then yeah that works.